testing/browser-functional/nightwatch.conf.js

```diff
@@ -30,14 +28,22 @@ module.exports = {
         },
 
         chrome: {
-            extends: 'selenium',
+            silent: true,
```

Comment:
I dont know about this extends selenium, are we sure we are not missing some functionality? or that extends selenium is the new selenium: {
                start_process: false,
            }?